lib/components.dart

```diff
@@ -28,4 +22,3 @@ export 'src/components/text_component.dart';
 export 'src/extensions/vector2.dart';
 export 'src/text_config.dart';
 export 'src/timer.dart';
```

Comment:
probably a terrible merge/rebase accident. sadly our linter doesn't seem to care for repeated export (imports work)